string need_name
string person_name
string IDtype
string rob_emotion
string person_emotion
float64 weight
string speech